examples/train_cifar100_with_xbm.py

```diff
@@ -86,7 +86,7 @@ def configure_loss(self) -> SimilarityLoss:
         return TripletLoss(mining="semi_hard")
 
     def configure_xbm(self) -> XbmConfig:
-        return XbmConfig()
+        return XbmConfig(start_iteration=100, buffer_size=1024)
```

Comment:
`start_iteration` is too small, isn't it?

Reply:
I kept it small to fail early in case of any problem, but we can also use the default value there if you prefer that one.

Reply:
I'd prefer to set such a value when the impact of xbm is valuable against training without it.
I'm not sure if we're comparing this to regular training or this example is just for demonstrating the interface.

If this example is for demonstration purposes only, then it should be written in docstrings/readme

Reply:
Reverted to the default value and also removed `mining` option in `argparse` as it's unused for now.

Btw, what do you mean with `docstrings/readme`?